src/SonataCoreBundle.php

```diff
@@ -146,10 +139,6 @@ public function registerFormMapping(): void
             'form.type_extension.form.data_collector',
         ];
 
-        if (class_exists(DescriptionFormTypeExtension::class)) {
-            $formTypes[] = 'nelmio_api_doc.form.extension.description_form_type_extension';
-        }
-
         FormHelper::registerFormExtensionMapping('form', $formTypes);
```

Comment:
I think everything related to FormHelper should get removed

Reply:
There wasn't any kind of comment so I wasn't sure :confused:

Reply:
@greg0ire the formHelper is used only to keep compat with old form types declaration, or has more uses?

Reply:
Not sure… maybe try removing it in a real app with form mapping disabled?

Reply:
found the depreciation, will remove it now.